### PR TITLE
Glossary: tidy ready for markdown (learn > see also)

### PR DESCRIPTION
**Pull request by hamishwillee**
As discussed in https://github.com/mdn/content/discussions/9008#discussioncomment-1345514 , the "Learn more" is inconsistent with the rest of the docs (which use *See also*) and the sub-headings division into general and technical learning is very artificial.

This PR
- Replaces all "Learn more" with See also
- Replaces Quick links with `See also` (or merging where needed)
- Removes unnecessary `rel="external"` on links
- General minor fixes.

### CODE REVIEW

**Review by teoli2003 — APPROVED** [review]

File: files/en-us/glossary/fork/index.html [hunk 1]
```diff
@@ -13,28 +13,17 @@
...
-<h3 id="Various_well-known_forks">Various "well-known" forks</h3>
+<h2 id="see_also">See also</h2>
 
 <ul>
- <li><a href="https://upload.wikimedia.org/wikipedia/commons/1/1b/Linux_Distribution_Timeline.svg">Linux distributions</a></li>
- <li><a href="https://nodejs.org/en/blog/announcements/foundation-v4-announce/">Node.js and io.js (which have been merged together back)</a></li>
- <li><a href="https://www.libreoffice.org/about-us/who-are-we/">LibreOffice, a fork of OpenOffice</a></li>
+  <li>{{interwiki("wikipedia", "Fork_(software_development)","Fork")}} on Wikipedia</li>
+  <li><a href="https://help.github.com/articles/fork-a-repo/">How to fork a GitHub repo</a> (fork as in a Git context)</li>
+  <li>{{Glossary("Fork")} (Glossary)</li>
```
Comment (teoli2003) [inline files/en-us/glossary/fork/index.html:21]:
```suggestion
  <li>{{Glossary("Fork")}} (Glossary)</li>
```

**Review by teoli2003 — CHANGES_REQUESTED** [review]
An error in a macro

File: files/en-us/glossary/fork/index.html (hunk 1 above)
Comment (hamishwillee) [reply files/en-us/glossary/fork/index.html]: Sharp eyes!

Comment (teoli2003) [reply files/en-us/glossary/fork/index.html]: Naah, I missed it. But the CI got it :-)

**Review by teoli2003 — APPROVED** [review]

**Pull request merged**